aunt-leahs-app/src/theme.js

```diff
@@ -0,0 +1,16 @@
+import { createMuiTheme } from "@material-ui/core";
+
+export const alpTheme = createMuiTheme({
+    typography: {
+        fontFamily: 'Roboto',
+    },
+    palette: {
+        primary: {
+            main: '#1AAE9F',
+        },
+        secondary: {
+            main: '#D3455B',
+        },
```

Comment:
Extra comma?

Reply:
The code still works with the extra comma, but I will remove it for code style reasons